### ISSUE TITLE
Add .idea/ files to .gitignore

### ISSUE THREAD

**Issue by 4n70w4** [docker-taiga/taiga#9]

**Comment by w1ck3dg0ph3r** [comment]
I don't believe this is necessary as there's no limited set of editors people use to edit those files and we can't add all of them to .gitignore.

**Comment by 4n70w4** [comment]
@w1ck3dg0ph3r 
So all editors is not needed. The idea editors is the industry standard.

Why did you immediately close the ticket, what about a free discussion?

**Comment by w1ck3dg0ph3r** [comment]
Sorry for being probably too harsh)

> The idea editors is the industry standard.

This very much depends on the specific industry we talking about, though. And the ability to use taiga to manage projects doesn't. 

So sorry, but I don't really see a point in adding .gitignore to the three file repo just to cater to the users of one specific editor. If you are using some specific editor, you can simply add relevant entries to your local .gitignore (and include the .gitignore itself).

**Comment by ctreton** [comment]
.idea folder is in my opinion a personal configuration and should be in your global .gitignore file.
But a .gitignore file could be helpful with the folder 'data' created by volume.
